tests/e2e/default/more-options-menu/online-user.wdio-spec.js

```diff
@@ -0,0 +1,119 @@
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const contactPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
+const reportPage = require('../../../page-objects/default/reports/reports.wdio.page');
+const placeFactory = require('../../../factories/cht/contacts/place');
+const reportFactory = require('../../../factories/cht/reports/generic-report');
+const personFactory = require('../../../factories/cht/contacts/person');
+const uuid = require('uuid').v4;
+const utils = require('../../../utils');
+
+const places = placeFactory.generateHierarchy();
+const clinic = places.get('clinic');
+const health_center = places.get('health_center');
+const district_hospital = places.get('district_hospital');
+
+const contact = personFactory.build({
+  _id: uuid(),
+  name: 'contact',
+  phone: '+12068881234',
+  place: health_center._id,
+  type: 'person',
+  parent: {
+    _id: health_center._id,
+    parent: health_center.parent
+  },
+});
+
+const patient = personFactory.build({
+  _id: uuid(),
+  parent: { _id: clinic._id, parent: { _id: health_center._id, parent: { _id: district_hospital._id }}}
+});
+
+const report = reportFactory.build({ form: 'home_visit', content_type: 'xml' }, { patient, submitter: contact });
+
+const sendMessage = async (message = 'Testing', phone = contact.phone) => {
```

Comment:
you can create a sms.js in the tests/utils folder. If you think this is something we're going to need again :)